DUnitX.TestFramework.pas

```diff
@@ -584,22 +602,6 @@   TDUnitX = class
     function GetReport: string;
   end;
 
-
-  ETestFrameworkException = class(Exception);
```

Comment:
This was key to getting the "comparable" exception to work.  When I investigated why those were being assigned to class vars on the Assert, I couldn't really figure out the point.  Moving them to another file to prevent circular ref worked and all tests passed, so I went with it...